targets/zephyr/src/Makefile

```diff
@@ -19,4 +19,4 @@ endif
 # Adding path for jerry script APIs
 ZEPHYRINCLUDE += -I$(JERRY_INCLUDE)
 
-obj-y += main-zephyr.o
+obj-y += main-zephyr.o zephyr_getline.o
```

Comment:
It similar matter to codestyle mentioned above -  zephyr_getline.o tries to follow zephyr's conventions, which uses underscores in filenames. I agree it's inconsistent, but it'a about same kind of inconsistency as, for example mbed port, which require C++ at all. I can change it if JerryScript maintainers between "ease of pulling updates from upstream" and "following JrS own conventions" choose the latter.